include/dsAudio.h

```diff
@@ -2063,6 +2063,30 @@ dsError_t  dsGetSecondaryLanguage(intptr_t handle, char* sLang);
  */
 dsError_t dsGetHDMIARCPortId(int *portId);
 
+/**
+* @brief Sets the Mixer Volume level for the given input
+*
+* This function sets the mixer volume level for either player/primary volume
+*
+* @param[in] handle  - Handle for the output Audio port
+* @param[in] aInput  - dsAudioInputPrimary / dsAudioInputSystem
+* @param[in] volume  - volume to be set (0 to 100)
+*
+* @return dsError_t                        - Status
+* @retval dsERR_NONE                       - Success
+* @retval dsERR_NOT_INITIALIZED            - Module is not initialised
+* @retval dsERR_INVALID_PARAM              - Parameter passed to this function is invalid
+* @retval dsERR_OPERATION_NOT_SUPPORTED    - The attempted operation is not supported; e.g: source devices
+* @retval dsERR_OPERATION_FAILED           - The attempted operation has failed
+*
+* @pre dsAudioPortInit() must be called before calling this API
+*
+* @warning  This API is Not thread safe
+*
+*
+*/
+dsError_t dsSetAudioMixerLevels (intptr_t handle, dsAudioInput_t aInput, int volume);
```

Comment:
Do we get the handle without calling dsGetAudioPort()? if not add dsGetAudioPort() to Precondition